--- /home/jack/.local/share/omarchy/config/uwsm/env	2026-06-13 14:06:38.856411966 -0700
+++ uwsm/env	2026-06-13 14:13:41.607303679 -0700
@@ -2,7 +2,7 @@
 
 # Ensure Omarchy bins are in the path
 export OMARCHY_PATH=$HOME/.local/share/omarchy
-export PATH=$OMARCHY_PATH/bin:$PATH:$HOME/.local/bin
+export PATH=$OMARCHY_PATH/bin:$PATH
 
 # Set default terminal and editor
 source ~/.config/uwsm/default
