--- /home/jack/.local/share/omarchy/config/hypr/hyprlock.conf	2026-03-01 07:40:53.037267600 -0800
+++ hypr/hyprlock.conf	2026-03-01 07:45:22.469685526 -0800
@@ -1,5 +1,5 @@
 source = ~/.config/omarchy/current/theme/hyprlock.conf
-
+ 
 general {
     ignore_empty_input = true
 }
@@ -26,7 +26,7 @@
     outer_color = $outer_color
     outline_thickness = 4
 
-    font_family = JetBrainsMono Nerd Font
+    font_family = CaskaydiaMono Nerd Font
     font_color = $font_color
 
     placeholder_text = Enter Password
