--- /home/jack/.local/share/omarchy/config/btop/btop.conf	2026-06-13 14:06:38.853975652 -0700
+++ btop/btop.conf	2026-06-12 13:52:19.480058910 -0700
@@ -1,4 +1,4 @@
-#? Config file for btop v.1.4.6
+#? Config file for btop v.1.4.7
 
 #* Name of a btop++/bpytop/bashtop formatted ".theme" file, "Default" and "TTY" for builtin themes.
 #* Themes should be placed in "../share/btop/themes" relative to binary or "$HOME/.config/btop/themes"
@@ -14,6 +14,11 @@
 #* Will force 16-color mode and TTY theme, set all graph symbols to "tty" and swap out other non tty friendly symbols.
 force_tty = false
 
+#* Option to disable presets. Either the default preset, custom presets, or all presets.
+#* "Off" All presets are enabled.
+#* "Default" preset is disabled.#* "Custom" presets are disabled.#* "All" presets are disabled.
+disable_presets = "Off"
+
 #* Define presets for the layout of the boxes. Preset 0 is always all boxes shown with default settings. Max 9 presets.
 #* Format: "box_name:P:G,box_name:P:G" P=(0 or 1) for alternate positions, G=graph symbol to use for box.
 #* Use whitespace " " as separator between different presets.
@@ -24,6 +29,9 @@
 #* Conflicting keys for h:"help" and k:"kill" is accessible while holding shift.
 vim_keys = true
 
+#* Disable all mouse events.
+disable_mouse = false
+
 #* Rounded corners on boxes, is ignored if TTY mode is ON.
 rounded_corners = true
 
@@ -56,11 +64,11 @@
 shown_boxes = "cpu mem net proc"
 
 #* Update time in milliseconds, recommended 2000 ms or above for better sample times for graphs.
-update_ms = 2000
+update_ms = 1000
 
 #* Processes sorting, "pid" "program" "arguments" "threads" "user" "memory" "cpu lazy" "cpu direct",
 #* "cpu lazy" sorts top process over time (easier to follow), "cpu direct" updates top process directly.
-proc_sorting = "cpu lazy"
+proc_sorting = "cpu direct"
 
 #* Reverse sorting order, True or False.
 proc_reversed = false
@@ -92,6 +100,9 @@
 #* (Linux) Filter processes tied to the Linux kernel(similar behavior to htop).
 proc_filter_kernel = false
 
+#* Should the process list follow the selected process when detailed view is open.
+proc_follow_detailed = true
+
 #* In tree-view, always accumulate child process resources in the parent process.
 proc_aggregate = false
 
@@ -208,6 +219,9 @@
 #* Example: "/mnt/media:100 /:20 /boot:1".
 io_graph_speeds = ""
 
+#* Swap the positions of the upload and download speed graphs. When true, upload will be on top.
+swap_upload_download = false
+
 #* Set fixed values for network graphs in Mebibits. Is only used if net_auto is also set to False.
 net_download = 100
 
@@ -250,7 +264,7 @@
 #* Horizontally mirror the GPU graph.
 gpu_mirror_graph = true
 
-#* Set which GPU vendors to show. Available values are "nvidia amd intel"
+#* Set which GPU vendors to show. Available values are "nvidia amd intel apple"
 shown_gpus = "nvidia amd intel"
 
 #* Custom gpu0 model name, empty string to disable.
